### ISSUE TITLE
An error while taken the offer, the 4th so far, fees charged 

### ISSUE THREAD

**Issue by eespn** [bisq-network/bisq#1148]
Hi, I am having this problem frequently, this is the forth time, I do not understand why you do not take a commission only after confirming the buy, I have already another ticket opened via email for the previous 3 cases.

![image](https://user-images.githubusercontent.com/35086537/34547848-edd68a60-f0b2-11e7-800e-d2960017b552.png)

I receive a message that there has been an error and a link to open a ticket in GitHUb or to open the log, now the only I can do is cancel, once I do that I get charged either way, it tells me that I have already funded the offer, that if I cancel the funds will be moved to the wallet again, but it didn't happen with the whole amount the last three times... so I have no hopes this time,

![image](https://user-images.githubusercontent.com/35086537/34547889-350c4b4a-f0b3-11e7-9201-51699f7e2a2f.png)

If I try t cancel and try to buy again it will not let me, the "Review: Take offer to buy Bitcoin" button is not functional any longer

![image](https://user-images.githubusercontent.com/35086537/34547924-73043de0-f0b3-11e7-8094-9a198ebd0c61.png)

This time, since it is the fourth time, I closed the tab without hitting cancel, hoping it will not charge me the deposit, and it worked (had to re-start Bisq for my balance to update), **BUT** still, a mysterious amount was withdrawn  "0.00205640", which is no small amount, it is over 30 USD

![image](https://user-images.githubusercontent.com/35086537/34548006-13a4916e-f0b4-11e7-90a2-f086ce1a9bcb.png)

As you can see from the image above, from the wallet an amount of 0.00205640 was withdraw, I can't understand why , can't make sense of it
 
Please assist to get this amount back, unfortunately have three additional cases before even worst than this one....,

Thanks!

**Comment by ManfredKarrer** [comment]
Can you send me the log file (account/backup -> button)? My email is manfred at bitsquare dot io.

Please contact @cbeams regarding the lost fee payments.

**Comment by eespn** [comment]
Thanks, I will be sending you the log soon after with code Ticket 1148 EESPN

**Comment by cbeams** [comment]
Issue moved to [bisq-network/support #5](https://github.com/bisq-network/support/issues/5) via [**ZenHub**](https://www.zenhub.com/)
